schemas/manifest.fbs

```diff
@@ -90,6 +90,9 @@ table ManifestV1 {
 
     // The URI of the object store dedicated specifically for WAL, if any.
     wal_object_store_uri: string;
+
+    // The recent seq that is safe for retention.
+    retention_min_seq: ulong;
```

Comment:
(bike shed brigade reporting for duty).

Nit: WDYT about naming it what it is, not what it's meant for. Something like `active_snapshot_min_seq`? And comment would be something like `// The minimum sequence number referenced across all active snapshots in the writer.

Reply:
love this, let me update it with `recent_snapshot_min_seq` 😁